README.md

```diff
@@ -142,7 +147,7 @@ python setup.py develop    # because no stable release yet
 
 see `pyproject.toml` file for full list of dependencies
 
-## <a name="running"></a> Running:
+## Running
 
 ### Verify Installation
```

Comment:
This also could be subject to moving to CONTRIBUTING page (see another note)

Reply:
I kept the information to ensure that MOABB is working well. From users' feedback in workshop, this is important.